app/src/components/posts/VideoPlayer.vue

```diff
@@ -23,20 +27,115 @@ onMounted(() => {
         },
         autoplay: false,
         preload: "auto",
+        controlBar: {
+            muteToggle: false,
+            volumePanel: false,
+            remainingTimeDisplay: false,
+        },
     };
 
     const player = videojs(playerElement.value!, options);
 
     player.poster(props.contentParent.image);
     player.src({ type: "application/x-mpegURL", src: props.contentParent.content[0].video });
+
+    // @ts-ignore - videojs-mobile-ui
```

Comment:
It can help to leave a little more readable comment here, like

```suggestion
    // @ts-ignore 2024-04-12 Typing of videojs-mobile-ui don't work
```